### ISSUE TITLE
feature request: sync predictive simulation settings from nightscout profile

### ISSUE THREAD

**Issue by jroth-haj** [NightscoutFoundation/xDrip#962]
sync predictive simulation settings from nightscout profile

**Comment by Navid200** [comment]
@jroth-haj Would you please close this issue if it has already been addressed?
Otherwise, it will help if you explain in more detail what the problem is that this new feature would address.
Considering the limited resources (developers and developer time) we have and the number of open issues, we have to prioritize.  "Nice-to-haves" will not get priority as long as we still have outstanding "critical" issues.

**Comment by jroth-haj** [comment]
Sorry, its so long ago.

**Comment by jroth-haj** [comment]
OK, I remember now.
Instead of Change the settings for carb ratio,... it would bei nice to geht this from the Nightscout Profile, If Nightscout ist used.

**Comment by Navid200** [comment]
@jroth-haj Would you please explain what you are proposing clearly.  If needed, please update the title as well.
Please make it as easy as possible for a developer to understand what you are proposing.

**Comment by jroth-haj** [comment]
@Navid200
In xDrip predictive Simulation settings are the Profile values Insulin
sensotivity and carb ratio. xDrip Schuld able to get this settings from the
Nightscout Profile and also reread this settings If the Nightscout Profile
hast changed.

Navid200 <notifications@github.com> schrieb am Fr., 26. Feb. 2021, 22:05:

**Comment by Navid200** [comment]
Would you please explain how those parameters would get to Nightscout in the first place?

**Comment by jroth-haj** [comment]
If a User has a Nightscout sync enabled, this values can bei read from Nightscout by enabling an new option "read Profile from Nightscout".
Only a one way sync from Nightscout to xDrip.

**Comment by Navid200** [comment]
Let me ask my question differently.  
Before you can read those values from Nightscout, those values need to be loaded into Nightscout.  
How are you doing to enter those parameters into Nightscout?

**Comment by jroth-haj** [comment]
If the User wants to sync from Nightscout, he ist putting the values directly in his in Nightscout Profile.

**Comment by Navid200** [comment]
I think I understand now.
The user has to enter the parameters into Nightscout.  And the user can enter the parameters into xDrip as well.  However, it would be convenient if the user could enter the information only once.  I suppose that's what you have been doing.
Please correct me if that is not correct.

I would like to point out that the xDrip code has become very complicated.  We are trying to categorize the issues into different categories to help the developers make progress in resolving them.

**Comment by Navid200** [comment]
I would like to bring to your attention a different issue:
https://github.com/NightscoutFoundation/xDrip/issues/813

It's basically someone stating that when the alarm goes off, he has no control over the volume.  He finds even the lowest level, being medium, too loud.  he is asking for a lower volume.
He has no work-around.  I guess he can stop using xDrip.  But, that's not a work-around.
Your issue is as important.  But, you do have a work-around.  You can manually enter the parameters into xDrip.

I would like to postpone your issue until we take care of the critical (without workaround) issues.  So, I will close the issue but label it as postponed.  So, it will not be confused with an issue that has been resolved or an issue that is not going to be resolved.  Rather an issue that is temporarily on hold.
This will allow us to reduce the number of open issue to give more visibility to the critical ones.

I hope this is acceptable to you and thanks for reporting this.

**Comment by jroth-haj** [comment]
I'm glad that you are checking all open issues.
If the number of open issues ist reduced, you can concertrate to it and you can create a roadmap for nee features or for a compleate redesign of the code.
That will be good for the future.
Yes, I accept your decision.
Thank you all for the great work!

**Comment by rpm5099** [comment]
I am just curious if this feature was ever implemented.  It's particularly useful for parents trying to keep up with changing carb ratios and insulin sensitivity.  Much appreciated.
